onboard/nucleo_bridge/LCMHandler.cpp

```diff
@@ -0,0 +1,188 @@
+#include "LCMHandler.h"
+
+void LCMHandler::init() {
+    //Creation of lcm bus
+    lcm_bus = new lcm::LCM();
+    if (!lcm_bus->good())
+    {
+        printf("LCM Bus not created\n");
+        exit(1);
+    } else {
+	printf("LCM Bus created\n");
+    }
+    
+    internal_object = new InternalHandler();
+    
+    //Subscription to lcm channels 
+    lcm_bus->subscribe("/ik_ra_control", &LCMHandler::InternalHandler::ra_closed_loop_cmd, internal_object);
+    lcm_bus->subscribe("/sa_closedloop_cmd", &LCMHandler::InternalHandler::sa_closed_loop_cmd, internal_object);
+
+    lcm_bus->subscribe("/ra_openloop_cmd", &LCMHandler::InternalHandler::ra_open_loop_cmd, internal_object);
+    lcm_bus->subscribe("/sa_openloop_cmd", &LCMHandler::InternalHandler::sa_open_loop_cmd, internal_object);
+
+    lcm_bus->subscribe("/gimbal_openloop_cmd", &LCMHandler::InternalHandler::gimbal_cmd, internal_object);
+    lcm_bus->subscribe("/hand_openloop_cmd", &LCMHandler::InternalHandler::hand_openloop_cmd, internal_object);
+    lcm_bus->subscribe("/foot_openloop_cmd", &LCMHandler::InternalHandler::foot_openloop_cmd, internal_object);
+    /*
+    lcmBus->subscribe("/ra_config_cmd", &LCMHandler::InternalHandler::ra_config_cmd, &internal_object);
+    lcm_bus.subscribe("/sa_config_cmd", &LCMHandler::InternalHandler::sa_config_cmd, internal_object);
+    lcm_bus.subscribe("/ra_zero_trigger", &LCMHandler::InternalHandler::ra_zero_trigger, &internal_object);
+    lcm_bus.subscribe("/sa_zero_trigger", &LCMHandler::InternalHandler::sa_zero_trigger, internal_object);
+    */

```

Comment:
remove

Reply:
This could be important for when we activate IK on the 2021 arm